python/mxnet/contrib/amp/amp.py

```diff
@@ -342,3 +349,320 @@ def unscale(optimizer_or_trainer):
+                                            mx_uint(len(conditional_op_names)),
+                                            mx_uint(len(excluded_sym_names)),
+                                            mx_uint(len(model_param_names)),
+                                            c_str_array(target_dtype_ops),
+                                            c_str_array(fp32_ops),
+                                            c_str_array(widest_dtype_ops),
+                                            c_str_array(conditional_op_names),
+                                            c_str_array(excluded_sym_names),
+                                            c_str_array(param_names),
+                                            c_str_array(param_vals),
+                                            c_str_array(model_param_names),
+                                            keys))
+    return Symbol(out)
+
+def convert_model(sym, arg_params, aux_params, target_dtype="float16", target_dtype_ops=None,
+                  fp32_ops=None, conditional_fp32_ops=None, excluded_sym_names=None):
+    """API for converting a model from FP32 model to a mixed precision model.
+    MXNet tries to convert the FP32 model to mixed precision model by adding
+    cast layers using amp_cast and amp_multicast operators which can be used for inference use cases.
+    The decision on which cast layer to add is based on hardcoded lists for Automatic Mixed Precision
+    in MXNet. These lists can be overridden by the user by providing their own lists
+    using : targe_precision_ops, fp32_ops, widest_precision_ops, conditional_fp32_ops
+
+    arg_params : dict
+        Dictionary of name to `NDArray`.
+    aux_params : dict
+        Dictionary of name to `NDArray`.
+    target_dtype : str
+        Currently only supports float16. The target dtype indicates to add cast layers
+        when possible so that lower precision computation can be leveraged.
+    target_dtype_ops : list of strs
+        Override the list of operator names casted to target_dtype.
+        If None, uses the framework's default list to be casted to target dtype.
+    fp32_ops : list of strs
+        Override the lists of operator names casted to FP32.
+        If None, uses the framework's default list to be casted to FP32.
+    widest_dtype_ops : list of strs
+        A list of op names provided by user which should run in widest precision among its inputs.
+        If None, uses the framework's default list of widest_precision_ops.
+    conditional_fp32_ops : list of (string, string, list of string)
+        Override the list of operators to be casted to FP32.
+        The format of the list is
+        (name of the function, name of the parameter,
+         list of values of the parameter that make the operator to be casted to
+        fp32)
+    excluded_sym_names : list of strs
+        A list of strings that represent the names of symbols that users want to exclude
+        from being quantized.
```

Comment:
I'm not sure if low precision is a kind of quantization, maybe change `quantized` to `low precision`?

Reply:
sure will change this.